script/2_PowerSystems_examples/serialize_data.jl

```diff
@@ -10,17 +10,19 @@
 # ### Dependencies
 # Let's use a dataset from the [tabular data parsing example](../../notebook/2_PowerSystems_examples/parse_matpower.ipynb)
 using SIIPExamples
+using Logging
+logger = configure_logging(console_level = Error, file_level = Info, filename = "ex.log")
 pkgpath = dirname(dirname(pathof(SIIPExamples)))
 include(joinpath(pkgpath, "test", "2_PowerSystems_examples", "parse_matpower.jl"))
 
 # ### Write data to a temporary directory
 
 folder = mktempdir()
 path = joinpath(folder, "system.json")
-@info "Serializing to $path"
+println("Serializing to $path")
 to_json(sys, path)
 
-filesize(path) / 1000000 #MB
+filesize(path) / (1024 * 1024) #MiB
```

Comment:
Tools like `ls` usually provide these sizes in base 2, not base 10. 1 Mebibyte = 2^20 bytes